### ISSUE TITLE
Trackball Controls not extending properly

### ISSUE THREAD

**Issue by nart4hire** [Tresjs/tres#1134]
### Describe the bug

Follow up for this [Discussion](https://github.com/orgs/Tresjs/discussions/755).

tl;dr: I tried extending Trackball Controls according to the documentation for extending Orbit Controls, however it didn't work. Confirmed to be a bug with Cientos, so I made this issue.

### Reproduction

https://stackblitz.com/edit/tresjs-minimal-reproduction-vev5jt?file=src%2Fcomponents%2FTheExperience.vue

### Steps to reproduce

Extend trackball controls like how you would with orbit controls as mentioned in the TresJS docs

### System Info

```shell
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.20.3 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.2.3 - /usr/local/bin/npm
    pnpm: 8.15.6 - /usr/local/bin/pnpm
  npmPackages:
    @tresjs/cientos: 3.9.0 => 3.9.0 
    @tresjs/core: 4.0.2 => 4.0.2 
    vite: ^5.3.2 => 5.3.2
```

### Used Package Manager

pnpm

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/cientos/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/cientos/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://cientos.tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/Tresjs/cientos/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
